restructuredtext_lint/test/test.py

```diff
@@ -1,11 +1,17 @@
 import os
+from subprocess import check_output, CalledProcessError
+import sys
 from unittest import TestCase
 
 import yaml
 
 import restructuredtext_lint
 
+
 __dir__ = os.path.dirname(os.path.abspath(__file__))
+valid_rst = os.path.join(__dir__, 'test_files', 'valid.rst')
+invalid_rst = os.path.join(__dir__, 'test_files', 'invalid.rst')
+rst_lint_path = os.path.join(__dir__, os.pardir, 'cli.py')
```

Comment:
TIL about `os.pardir`. From the docs it looks like it's `..` in all cases but I guess they like future proofing.

https://docs.python.org/2/library/os.html#os.pardir